components/instructions/programs/system.tsx

```diff
@@ -46,5 +47,27 @@ export const SYSTEM_INSTRUCTIONS = {
         )
       },
     },
+    3: {
+      name: 'System Program - Create Account With Seed',
+      accounts: [
+        'Obligation',
+        'Lending Market Account',
+        'Obligation Owner',
+        'Sysvar: Clock',
+        'Rent Program',
+        'Token Program',
+      ],
+      getDataUI: (
+        _connection: Connection,
+        _data: Uint8Array,
+        _accounts: AccountMetaData[]
+      ) => {
+        return (
+          <>
+            <p>No data</p>
```

Comment:
`No instruction data` ?

Reply:
here there is no encoded data since it is just an account creation